toolbox/autogenerated/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+cmake_policy(SET CMP0048 NEW)
+project(ClockServer LANGUAGES CXX VERSION 0.1)
```

Comment:
I don't think it is needed to create different projects. A target should be sufficient (? @traversaro ?)

Reply:
Here I preferred starting from scratch with a new project in order to have them as much independent as possible. Suggestions are welcome.

Reply:
The `project` CMake command in the end is not really something special, it just defines a bunch of variables. However, I see two possible problems here: 
* It is not clear what it is the version declared here, 
* A non-generated file in a directory `autogenerated` ? I typically try to keep only auto-generated files in a directory called `autogenerated`, but perhaps it is just a personal taste.

Reply:
This folder contains autogenerated code, but is copied manually here. I already discussed last week with @francesco-romano your same concern about the name of this folder. What we would need is the installed thrift file, that currently nor yarp and gazebo-yarp-plugins install. We agreed on waiting one of those projects (yarp if I recall well) to handle this installation in order to clean this folder and let it become a real autogenerated directory.

For what concerns the CMake file, with this logic is indeed useless having a separate project , it will become a target as soon as this autogeneration will work as expected. For the time being if you don't have any concern I would leave this as it is and open an issue for future reference.